src/app/component/MyAccountMyWishlist/MyAccountMyWishlist.component.js

```diff
@@ -11,20 +11,93 @@
 
 import { PureComponent } from 'react';
 import PropTypes from 'prop-types';
+import { ProductType } from 'Type/ProductList';
+import ProductCard from 'Component/ProductCard';
+import WishlistItem from 'Component/WishlistItem';
 import './MyAccountMyWishlist.style';
 
-class MyAccountMyWishlist extends PureComponent {
+export default class MyAccountMyWishlist extends PureComponent {
     static propTypes = {
-        // TODO: implement prop-types
+        isLoading: PropTypes.bool,
+        removeAll: PropTypes.func,
+        addAllToCart: PropTypes.func,
+        isWishlistEmpty: PropTypes.bool,
+        wishlistItems: PropTypes.objectOf(ProductType).isRequired
     };
 
+    static defaultProps = {
+        isLoading: false,
+        removeAll: () => {},
+        addAllToCart: () => {},
+        isWishlistEmpty: false
+    };
+
+    renderNoProductsFound = () => (
+        <div block="MyAccountMyWishlist" elem="NoProducts">
+            <h3>Please add products to wishlist first!</h3>
```

Comment:
Please add translation. Please format this text better, do not center it, make it less bold, just a simple paragraph.